core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -1028,6 +1028,10 @@ class ControllerApis(
   def handleDescribeCluster(request: RequestChannel.Request): CompletableFuture[Unit] = {
```

Comment:
Is this request only ever originated from admin client?

Reply:
currently, yes